src/styles/_catalog.scss

```diff
@@ -1,22 +1,28 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(4, minmax(200px, 1fr));
```

Comment:
Now gap between the cards is not 48px but much more, please try to fix it
![image](https://github.com/mate-academy/layout_catalog_grid/assets/54936044/5bb3c4fe-13f2-41ab-881f-11e7029283b5)